examples/tsc/main.rs

```diff
@@ -0,0 +1,170 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+#[macro_use]
+extern crate lazy_static;
+#[macro_use]
+extern crate log;
+#[macro_use]
+extern crate futures;
+#[macro_use]
+extern crate serde_json;
+
+#[path = "../../src/ansi.rs"]
+mod ansi;
+#[path = "../../src/compiler.rs"]
+pub mod compiler;
+#[path = "../../src/deno_dir.rs"]
+pub mod deno_dir;
+#[path = "../../src/errors.rs"]
+pub mod errors;
+#[path = "../../src/flags.rs"]
+pub mod flags;
+#[path = "../../src/fs.rs"]
+mod fs;
+#[path = "../../src/http_body.rs"]
+mod http_body;
+#[path = "../../src/http_util.rs"]
+mod http_util;
+#[path = "../../src/isolate.rs"]
+pub mod isolate;
+#[path = "../../src/js_errors.rs"]
+pub mod js_errors;
+#[path = "../../src/libdeno.rs"]
+pub mod libdeno;
+#[path = "../../src/modules.rs"]
+pub mod modules;
+#[path = "../../src/msg.rs"]
+pub mod msg;
+#[path = "../../src/msg_util.rs"]
+pub mod msg_util;
+#[path = "../../src/ops.rs"]
+pub mod ops;
+#[path = "../../src/permissions.rs"]
+pub mod permissions;
+#[path = "../../src/repl.rs"]
+mod repl;
+#[path = "../../src/resolve_addr.rs"]
+pub mod resolve_addr;
+#[path = "../../src/resources.rs"]
+pub mod resources;
+#[path = "../../src/snapshot.rs"]
+pub mod snapshot;
+#[path = "../../src/tokio_util.rs"]
+mod tokio_util;
+#[path = "../../src/tokio_write.rs"]
+mod tokio_write;
+#[path = "../../src/version.rs"]
+pub mod version;
+#[path = "../../src/workers.rs"]
+pub mod workers;
```

Comment:
Hrm, I think it would be good to import `src/` as a whole crate ... not sure how to do that offhand